requirements.txt

```diff
@@ -5,3 +5,7 @@ django-heroku==0.3.*
 django-crispy-forms==1.8.*
 python-dotenv==0.10.*
 whitenoise==4.1.4
+djangorestframework
```

Comment:
We will use DRF on our project?

Reply:
Like, by a ton easier.

Reply:
All right, please introduce DRF to the team. 
Do we need to make changes to old APIs like logout? If so, we need to add an issue to the next Sprint.